lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -210,7 +210,7 @@ def test_send_to_all(self):
 
         test_email = {
             'action': 'Send email',
-            'send_to': 'all',
+            'send_to': '["myself", "staff", "learners"]',
```

Comment:
I think that if the idea is to send to all except cohort, the name of this test or at least a docstring should reflect that.